src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -486,27 +487,46 @@ def add_digital_object(self, parent_archival_object, dashboard_uuid, title="", i
             "notes": [{
                 "jsonmodel_type": "note_digital_object",
                 "type": "originalsloc",
-                "content": [dashboard_uuid]
+                "content": [dashboard_uuid],
+                "publish": True,
             }],
             "restrictions": restricted,
             "subjects": parent_record['subjects'],
             "linked_agents": parent_record['linked_agents'],
         }
 
+        i = 0
+        for pnote in parent_record["notes"]:
+            content = []
+            for subnote in pnote["subnotes"]:
+                content.append(subnote["content"])
+
+            new_object["notes"].append({
+                "jsonmodel_type": "note_digital_object",
+                "type": pnote["type"],
```

Comment:
It would be better to fetch the array, I think, but the funding to work on this feature has run out.  I will ask the client if they are interested in optimizing this, for now I vote for it to stay as a hardcoded array.